src/model_specific/mom6/common_mom6.f90

```diff
@@ -223,9 +188,8 @@ SUBROUTINE set_common_oceanmodel
   ! Close the grid files:
   !-----------------------------------------------------------------------------
   call check( NF90_CLOSE(ncid) )
-!!call check( NF90_CLOSE(ncid1) )
   call check( NF90_CLOSE(ncid2) )
-! call check( NF90_CLOSE(ncid3) )
+  call check( NF90_CLOSE(ncid3) )
```

Comment:
rename to `ncid1` and `ncid2` remove `ncid3`

Reply:
We have three input files here. ncid for file 1, ncid2 for file 2. And ncid3 for file 3. Do you still want me to change? Or change ncid to ncid1

Reply:
Just name them: `1, 2, 3`

Reply:
@sanAkel Done. removed `gridfile` and start to use `gridfile1/2/3` with corresponding `ncid1/2/3`